carsus/io/output/base.py

```diff
@@ -485,6 +600,36 @@ def lines_prepared(self):
 
         return lines_prepared
 
+    @property
+    def collisions_prepared(self):
+        """
+            Prepare the DataFrame with electron collisions for TARDIS
+            Returns
+            -------
+            collisions_prepared : pandas.DataFrame
+                DataFrame with:
+                    index: atomic_number, ion_number, level_number_lower, level_number_upper;
+                    columns: e_col_id, delta_e, g_ratio, [collision strengths].
```

Comment:
This is not a collision strength! See previous comment.
We should find a better name for this quantity.

Reply:
Done.